src/applications/mhv/secure-messaging/components/MessageList/FolderHeader.jsx

```diff
@@ -56,6 +56,7 @@ const FolderHeader = props => {
       </h1>
       <>{handleFolderDescription()}</>
       {folder.folderId === Folders.INBOX.id && <ComposeMessageButton />}
+      {folder.folderId === Folders.SENT.id && <ComposeMessageButton />}

```

Comment:
We may want to confirm with UCD if this button only appears on Inbox and Sent folders. If it should be there on any folder, then this condition may not be needed

Reply:
You're right it appears to be on every folder except trash in Sketch.  I'll adjust.